lmdeploy/model.py

```diff
@@ -492,6 +492,44 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'llama2'
 
 
+@MODELS.register_module(name='llama3')
+class Llama3(BaseChatTemplate):
+    """Chat template of LLaMA3 model."""
+
+    def __init__(self,
+                 system='<|start_header_id|>system<|end_header_id|>\n\n',
```

Comment:
We should match with https://github.com/meta-llama/llama3/blob/359887376f0aaf30e433f23e25df858d8c2a9833/llama/tokenizer.py#L38-L97

Reply:
Which part did not match?

Reply:
Seems the tokenizer of llama3 does not add bos token itself. I will update it.